### PR TITLE
update intel models index

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
- [x] remove odd info from index table
- [x] an attempt to made file more compact
- [x] add `accuracy` tab to the table, to make it consistent with public models index

### CODE REVIEW

File: models/intel/index.md [hunk 1]
```diff
@@ -4,9 +4,8 @@ OpenVINO&trade; toolkit provides a set of pre-trained models
 that you can use for learning and demo purposes or for developing deep learning
 software. Most recent version is available in the [repo on Github](https://github.com/openvinotoolkit/open_model_zoo).
 
-The models can be downloaded via Model Downloader
-(`<OPENVINO_INSTALL_DIR>/deployment_tools/open_model_zoo/tools/downloader`).
-They can also be downloaded manually from [01.org](https://download.01.org/opencv).
+The models can be downloaded via Model Downloader and other automation tools.
```
Comment (IRDonch) [inline models/intel/index.md:7]: That's not really true. You don't use _other_ tools to download models.

File: models/intel/index.md [hunk 2]
```diff
@@ -4,9 +4,8 @@ OpenVINO&trade; toolkit provides a set of pre-trained models
...
+Review the tools description in the file `<OPENVINO_INSTALL_DIR>/deployment_tools/open_model_zoo/tools/downloader/README.md` for details.
```
Comment (IRDonch) [inline models/intel/index.md:8]: You should use a relative link instead.

File: models/intel/index.md [hunk 3]
```diff
@@ -18,74 +17,73 @@ detect the same types of objects (for example, `face-detection-adas-0001` and
...
-| [yolo-v2-tiny-ava-0001](./yolo-v2-tiny-ava-0001/README.md)                                                                                               | 6.975                | 15.12      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-ava-sparse-30-0001](./yolo-v2-tiny-ava-sparse-30-0001/README.md)                                                                 | 6.975                | 15.12      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-ava-sparse-60-0001](./yolo-v2-tiny-ava-sparse-60-0001/README.md)                                                                 | 6.975                | 15.12      |       |      X  | X        | X     |                |         |
-| [yolo-v2-tiny-vehicle-detection-0001](./yolo-v2-tiny-vehicle-detection-0001/README.md)                                                     | 5.424                | 11.229     |       |         | X        |       |                |         |
-
+| Model Name | Accuracy | Complexity (GFLOPs) | Size (Mp) |
```
Comment (IRDonch) [inline models/intel/index.md:20]: "Accuracy" by itself doesn't mean much; it should name the specific metric that is used.

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: models/intel/index.md (hunk 2 above)
Comment (vladimir-dudnik) [reply models/intel/index.md]: changed

File: models/intel/index.md (hunk 1 above)
Comment (vladimir-dudnik) [reply models/intel/index.md]: changed

File: models/intel/index.md (hunk 3 above)
Comment (vladimir-dudnik) [reply models/intel/index.md]: accuracy column removed

**Review by IRDonch — APPROVED** [review]

**Pull request merged**